recipe/meta.yaml

```
@@ -26,7 +26,7 @@ requirements:
     - cryptography
     - graphene >=2.1,<3
     - jinja2 ==2.11.0,<2.12
-    - metomi-isodatetime ==1!2.0.1
+    - metomi-isodatetime ==1!2.0.2, <1!2.1.0
```

Should that be greater equals?

```suggestion
    - metomi-isodatetime >=1!2.0.2, <1!2.1.0
```

https://github.com/cylc/cylc-flow/blob/8b25ae063cfd3273447ec2339efcb83417d721e2/setup.py#L51

---

Ah, crap. Yes.